integrations/transformers/trainer_qa.py

```diff
@@ -24,8 +27,9 @@
     import datasets
 
 if is_torch_tpu_available():
-    import torch_xla.core.xla_model as xm
-    import torch_xla.debug.metrics as met
+    import torch_xla.core.xla_model as xm  # noqa: F401
+    import torch_xla.debug.metrics as met  # noqa: F401
```

Comment:
I am not sure if these two lines are necessary. In case they are I added an ignore not used.